jcloud/api.py

```diff
@@ -410,32 +427,71 @@ async def logs(args):
         show_lines=True,
     )
+    console = Console()
+    if args.resource == Resources.Flow:
+        name = 'gateway' if args.gateway else f'executor {args.executor}'
+        print(f'Fetching the logs for {name} of the Flow: [green]{args.flow}[/green]')
 
-    name = 'gateway' if args.gateway else f'executor {args.executor}'
-    print(f'Fetching the logs for {name} of the Flow: [green]{args.flow}[/green]')
+        if args.gateway:
+            logs = await CloudFlow(flow_id=args.flow).logs()
+        else:
+            logs = await CloudFlow(flow_id=args.flow).logs(args.executor)
+
+        for pod, pod_logs in logs.items():
+            with console.status(
+                f'[bold]Displaying logs of pod [green]{pod}[/green]...'
+            ):
+                _pod_id_row = add_table_row_fn(_t, 'POD_ID', center_align(pod))
+
+                _pod_logs_lines = '\n'.join(pod_logs.split('\n'))
+                _pod_logs_row = add_table_row_fn(
+                    _t,
+                    'Logs',
+                    Syntax(
+                        _pod_logs_lines,
+                        lexer='vctreestatus',
+                        line_numbers=1,
+                        code_width=90,
+                    ),
+                )
 
-    if args.gateway:
-        logs = await CloudFlow(flow_id=args.flow).logs()
+                for fn in [_pod_id_row, _pod_logs_row]:
+                    fn()
+                console.print(_t)
     else:
-        logs = await CloudFlow(flow_id=args.flow).logs(args.executor)
-
-    console = Console()
-    for pod, pod_logs in logs.items():
-        with console.status(f'[bold]Displaying logs of pod [green]{pod}[/green]...'):
-            _pod_id_row = add_table_row_fn(_t, 'POD_ID', center_align(pod))
-
-            _pod_logs_lines = '\n'.join(pod_logs.split('\n'))
-            _pod_logs_row = add_table_row_fn(
+        logs = await CloudFlow(flow_id=args.flow).job_logs(args.name)
+        with console.status(
+            f'[bold]Displaying logs of job [green]{args.name}[/green]...'
+        ):
+            _job_id_row = add_table_row_fn(_t, 'JOB_NAME', center_align(args.name))
+            _job_logs_lines = '\n'.join(logs.split('\n'))
+            _job_logs_row = add_table_row_fn(
                 _t,
                 'Logs',
                 Syntax(
-                    _pod_logs_lines,
+                    _job_logs_lines,
                     lexer='vctreestatus',
                     line_numbers=1,
                     code_width=90,
                 ),
             )
-
-            for fn in [_pod_id_row, _pod_logs_row]:
+            for fn in [_job_id_row, _job_logs_row]:
                 fn()
             console.print(_t)
+
+
+@asyncify
+async def create(args):
+    if args.resource == Resources.Job:
+        return await CloudFlow(flow_id=args.flow).create_job(
+            args.name, args.image_name, args.timeout, args.backofflimit, args.entrypoint
+        )
+    else:
```

Comment:
here should have a default conditional output that if the `args.resources` is invalid, should tell user your input is invalid instead of using secret. Or it will confuse user.

Reply:
if the user sets the resource to anything other than the three accepted ones the parser will give an error.